build-tools-internal/src/main/resources/forbidden/jdk-deprecated.txt

```diff
@@ -9,6 +9,7 @@
 # This file contains all public, deprecated API signatures in Java version 17 (extracted from build 17).
 
 @ignoreMissingClasses
+@ignoreUnresolvable
```

Comment:
How granular is this? Does this apply to _all_ unresolved methods or just the ones in this file? Can we be more granular here and just ignore these known removed methods?

Also, do we know which third party library is using these removed methods? Can we update to a version that's fixed the issue?